core/src/mindustry/world/blocks/units/Reconstructor.java

```diff
@@ -114,6 +114,13 @@ public int getMaximumAccepted(Item item){
             return capacities[item.id];
         }
 
+        @Override
+        public void overwrote(Seq<Building> builds){
+            if(builds.first().block == block){
```

Comment:
Can be empty.

```suggestion
            if(builds.any() && builds.first().block == block){
```

Reply:
u sure? https://github.com/Anuken/Mindustry/blob/b63501b1c7cea96146b6664ba66b3f748bb71ddd/core/src/mindustry/world/blocks/ConstructBlock.java#L78

Reply:
Hmm, didn't see that.